pom.xml

```diff
@@ -338,9 +338,9 @@
             <systemPath>${basedir}/lib/Vampire.jar</systemPath>
         </dependency>
         <dependency>
-            <groupId>us.myles</groupId>
-            <artifactId>ViaVersion</artifactId>
-            <version>4.7.0</version>
+            <groupId>com.viaversion</groupId>
+            <artifactId>viaversion-api</artifactId>
```

Comment:
this should not be changed to `-api`, the jar included is the spigot plugin not an api jar